lumigator/backend/backend/tracking/mlflow.py

```diff
@@ -247,7 +248,7 @@ def get_workflow(self, workflow_id: str) -> WorkflowDetailsResponse | None:
             parameters=self._compile_parameters(all_job_ids),
         )
         # Currently, only compile the result json artifact if the workflow has succeeded
-        if workflow_details.status != WorkflowStatus.SUCCEEDED:
+        if workflow_details.status != WorkflowStatus.SUCCEEDED.value:
```

Comment:
Hmm. this doesn't seem like a needed change, right? workflow_details.status is WorkflowStatus type already, so why compare status with status.value? it's not clear.

Reply:
This is due to how the record was being stored in the label. Somehow it was being stored inconsistently as `status` or `WorkflowStatus.STATUS`. I've tweaked the code to always use the enum value in de/serialisation.

Reply:
Tweaked in main, aligned with main 👍